app/app.go

```diff
@@ -17,7 +17,13 @@ func init() {
 	auth = fb.Auth()
 
 	// auth server comes with CORS included
-	http.Handle("/", auth.Server(customClaims))
+	http.Handle("/", auth.Server(customClaims,
+		// Custom AllowedOrigins
+		firebase.ServerAllowedOrigins([]string{"*"}),
+		// Custom URI for verifyHandler
+		firebase.ServerVerifyURI("/customVerify"),
```

Comment:
I like showing how they are set, but could they be set back to the default string even though it seems redundant - the Polymer auth client lib uses this deployed example for it's demo.